target/linux/imx6/image/Makefile

```diff
@@ -235,4 +235,38 @@ define Device/apalis
 endef
 TARGET_DEVICES += apalis
 
+define Device/xiaomi_gateway-lumi
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Lumi Gateway
```

Comment:
This is overwritten, so you can drop it here.

Reply:
Squashed into a single node so it is not relevant anymore